rxtools/source/features/NandDumper.c

```diff
@@ -26,17 +26,17 @@ int NandSwitch(){
 	ConsoleShow();
 	while (true) {
         u32 pad_state = InputWait();
-		if(pad_state & BUTTON_X)    return  0;
-		if(pad_state & BUTTON_Y)    return  1;
-		if(pad_state & BUTTON_B) 	return -1;
+		if(pad_state & BUTTON_X) return SYS_NAND;
+		if(pad_state & BUTTON_Y) return EMU_NAND;
+		if(pad_state & BUTTON_B) return UNK_NAND;
     }
 }
 
 void NandDumper(){
 	File myFile;
-	int isEmuNand = checkEmuNAND() ? NandSwitch() : 0;
-	if(isEmuNand == -1) return;
-
+	int isEmuNand = SYS_NAND;
+	if(checkEmuNAND() && (isEmuNand = NandSwitch()) == UNK_NAND) return;
```

Comment:
@dukesrg I forget its function. Oh, and i suggest you to use your enum definitions.
Switch a SYS_NAND=1, minus it to 0, and compare -- really confused me a little. sorry.